src/MahApps.Metro.sln

```diff
@@ -36,10 +36,10 @@ Global
 		{D9ACF0D0-A178-4C91-BD23-D013CEA763A0}.Debug|Any CPU.Build.0 = Debug|Any CPU
 		{D9ACF0D0-A178-4C91-BD23-D013CEA763A0}.Release|Any CPU.ActiveCfg = Release|Any CPU
 		{D9ACF0D0-A178-4C91-BD23-D013CEA763A0}.Release|Any CPU.Build.0 = Release|Any CPU
-		{40048BCF-D1C7-46CC-B781-05718BE25BFC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
-		{40048BCF-D1C7-46CC-B781-05718BE25BFC}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{40048BCF-D1C7-46CC-B781-05718BE25BFC}.Release|Any CPU.ActiveCfg = Release|Any CPU
-		{40048BCF-D1C7-46CC-B781-05718BE25BFC}.Release|Any CPU.Build.0 = Release|Any CPU
+		{31D47FA4-9A93-4385-A326-76F189C02440}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{31D47FA4-9A93-4385-A326-76F189C02440}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{31D47FA4-9A93-4385-A326-76F189C02440}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{31D47FA4-9A93-4385-A326-76F189C02440}.Release|Any CPU.Build.0 = Release|Any CPU
 	EndGlobalSection
```

Comment:
Are these changes really necessary?

Reply:
Not necessary. It was added by VS automatically.